pkg/plugins/datasource_plugin_wrapper_v2.go

```diff
@@ -1,99 +0,0 @@
-package plugins
-
-import (
-	"context"
-	"fmt"
-
-	"github.com/grafana/grafana-plugin-sdk-go/backend"
-	"github.com/grafana/grafana-plugin-sdk-go/backend/grpcplugin"
-	"github.com/grafana/grafana-plugin-sdk-go/genproto/pluginv2"
-	"github.com/grafana/grafana/pkg/components/simplejson"
-	"github.com/grafana/grafana/pkg/infra/log"
-	"github.com/grafana/grafana/pkg/models"
-	"github.com/grafana/grafana/pkg/plugins/adapters"
-	"github.com/grafana/grafana/pkg/services/oauthtoken"
-)
-
-func newDataSourcePluginWrapperV2(log log.Logger, pluginId, pluginType string, dataClient grpcplugin.DataClient) *DatasourcePluginWrapperV2 {
-	return &DatasourcePluginWrapperV2{DataClient: dataClient, logger: log, pluginId: pluginId, pluginType: pluginType}
-}
-
-type DatasourcePluginWrapperV2 struct {
-	grpcplugin.DataClient
-	logger     log.Logger
-	pluginId   string
-	pluginType string
-}
-
-func (tw *DatasourcePluginWrapperV2) Query(ctx context.Context, ds *models.DataSource, query DataQuery) (DataResponse, error) {
-	instanceSettings, err := adapters.ModelToInstanceSettings(ds)
-	if err != nil {
-		return DataResponse{}, err
-	}
-
-	if query.Headers == nil {
-		query.Headers = make(map[string]string)
-	}
-
-	if oauthtoken.IsOAuthPassThruEnabled(ds) {
```

Comment:
We wanna keep this stuff right? Or have I missed where this is kept in the diff? 🤔

Reply:
Good catch :pray: Fixed